site/en/docs/privacy-sandbox/fledge/index.md

```diff
@@ -6,16 +6,20 @@ subhead: >
 description: >
   FLEDGE satisfies remarketing use cases, but is designed so it cannot be used by third parties to track user browsing behaviour across sites. The API enables on-device "auctions" by the browser, to choose relevant ads provided by websites the user has previously visited.
 date: 2021-05-18
-updated: 2021-09-10
+updated: 2021-09-16
 authors:
   - samdutton
 ---
 
-<!--lint disable no-smart-quotes-->
-
 {% Aside %}
-FLEDGE is the first experiment to be implemented in Chromium within the 
-[TURTLEDOVE](https://github.com/WICG/turtledove) family of proposals.
+FLEDGE can be tested in Chrome from versions 91.0.4472.49 or 92.0.4493.0 with the following flags enabled:
+
+`--enable-features=FledgeInterestGroups,FledgeInterestGroupAPI`
+
+For the best debugging experience, we recommend using the latest [Chrome Canary](https://www.google.com/chrome/canary).
+
+* The most recent [WICG call minutes](https://github.com/WICG/turtledove/blob/main/meetings/2021-05-12-FLEDGE-call-minutes.md#agenda) provides more detail about what is and is not supported in the current implementation.
```

Comment:
Could this be expanded and clarified a little, e.g. something like

This is the in-progress version of FLEDGE for early testing, so it should not be considered feature complete or indicative of the final implementation. Progress and status are discussed in the regular WICG meetings. The [minutes for the 2021/05/12](https://github.com/WICG/turtledove/blob/main/meetings/2021-05-12-FLEDGE-call-minutes.md#:~:text=Chrome%E2%80%99s%20FLEDGE%20implementation) call provide detail on what is and is not supported in the current implementation.

Reply:
That's much better — thanks.